superset-frontend/src/SqlLab/main.less

```diff
@@ -436,6 +436,10 @@ div.tablePopover {
   display: inline-block;
 }
 
+.QueryTable .ant-btn {
+  position: static;
```

Comment:
I couldn't find this.. where on the page is it?

Reply:
<img width="1060" alt="Screen Shot 2021-05-10 at 11 00 18 AM" src="https://user-images.githubusercontent.com/48933336/117680170-ed4a5780-b17e-11eb-8230-59f9c111dae6.png">


And it is for the admin and superset buttons 
<img width="1665" alt="Screen Shot 2021-05-10 at 11 00 36 AM" src="https://user-images.githubusercontent.com/48933336/117680264-fdfacd80-b17e-11eb-9b70-85140fd93771.png">